packages/airnode-deployer/src/utils/logger.ts

```diff
@@ -36,15 +36,25 @@ export function fail(text: string) {
 }
 
 export function warn(text: string) {
+  const currentOra = getSpinner();
+  if (currentOra.isSpinning) {
+    currentOra.clear();
+    currentOra.frame();
+  }
   oraInstance().warn(text);
 }
 
 export function info(text: string) {
+  const currentOra = getSpinner();
+  if (currentOra.isSpinning) {
+    currentOra.clear();
+    currentOra.frame();
+  }
   oraInstance().info(text);
 }
 
 export function debug(text: string) {
-  if (debugModeFlag) oraInstance().info(text);
+  if (debugModeFlag) info(text);
```

Comment:
These changes are to allow printing a line to the console while the `ora` spinner is still spinning. WIthout these the spinner will overwrite any logs.